src/main/java/tk/shanebee/hg/game/GamePlayerData.java

```diff
@@ -305,6 +307,20 @@ public void join(Player player, boolean command) {
                 freeze(player);
                 kills.put(player, 0);
 
+                ItemStack bed = new ItemStack(Material.RED_BED, 1);
```

Comment:
This item should probably be configurable in the config.
I just have a small feeling once this is added, people will ask for a config option